osdep/subprocess.c

```diff
@@ -37,3 +33,17 @@ const char *mp_subprocess_err_str(int num)
     default:                            return "unknown";
     }
 }
+
+void mp_subprocess(struct mp_log *log,
+                   struct mp_subprocess_opts *opts,
+                   struct mp_subprocess_result *res)
+{
+    mp_verbose(log, "Starting subprocess: [%s", opts->args[0]);
+    char **arg = &opts->args[1];
+    while(*arg)
+        mp_verbose(log, ", %s", *arg++);
+    mp_verbose(log, "]\n");
+    mp_subprocess2(opts, res);
+    if (res->error < 0)
+        mp_err(log, "Subprocess failed: %s\n", mp_subprocess_err_str(res->error));
```

Comment:
don't think this should be an error.
if a script just wants to probe for an utility it would be perfectly fine to just try running `foobar --version`.

Reply:
The error print for scripts was already there. Not sure we should change that.